src/assessments/readability/passiveVoiceAssessment.js

```diff
@@ -23,7 +23,6 @@ var calculatePassiveVoiceResult = function( passiveVoice, i18n ) {
 	var recommendedValue = 10;
 	var passiveVoiceURL = "<a href='https://yoa.st/passive-voice' target='_blank'>";
 	var hasMarks;
-
```

Comment:
Is there a particular reason for removing this newline?

Reply:
Not really, just as for swapping "es" and "fr".

Reply:
Reverted to the original code